Make modify volume interval configurable

adolsalamanca opened the pull request:
We're having problems related to race conditions when several changes are applied to volumes. We don't want those changes to be applied independently, so we're extracting this param to a driverOption to make it configurable from the outside.

**Issue**
Addresses the need described in [this issue](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/1911)

minkimipt reviewed (approved):
we need to build an image with this one and replace the original 1.25 image with it here https://github.com/timescale/savannah-infra/pull/2274/commits/3aba59ff5bcfb3d033cfe2636e08ac237a925efd

adolsalamanca commented:
> we need to build an image with this one and replace the original 1.25 image with it here [timescale/savannah-infra@3aba59f](https://github.com/timescale/savannah-infra/commit/3aba59ff5bcfb3d033cfe2636e08ac237a925efd)

That's correct, CI has been tweaked for that, I've been doing some tests with a custom tag

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/timescale/aws-ebs-csi-driver?pullRequest=8) <br/>All committers have signed the CLA.

andrewcharlton reviewed (approved):
:+1:

adolsalamanca committed:
Make modify volume interval configurable

We're having problems related to race conditions when several changes
are applied to volumes. We don't want those changes to be applied
independently, so we're extracting this param to a driverOption to make it
configurable from the outside.

adolsalamanca committed:
expose modifyVolumeConfigurable in chart
